roles/autofs/tasks/main.yml

```diff
@@ -22,7 +22,12 @@
     - configuration
 
 - name: ensure mountpoint exists
-  file: path={{ autofs_mount }} state=directory
+  file: 
+    path: "{{ autofs_mount }}"
+    state: directory
+    owner: "root"
+    group: "root"
+    mode: "0755"
```

Comment:
Is this mode value just the default? It does not appear in the previous main.yml